tests/cloud/useCKEditorCloud.test.tsx

```diff
@@ -3,21 +3,21 @@
  * For licensing, see LICENSE.md.
  */
 
-import { beforeEach, describe, expect, it } from 'vitest';
+import { beforeEach, describe, expect, expectTypeOf, it } from 'vitest';
 import { renderHook, waitFor, act } from '@testing-library/react';
 
 import type { CKEditorCloudConfig } from '@ckeditor/ckeditor5-integrations-common';
 import { removeAllCkCdnResources } from '@ckeditor/ckeditor5-integrations-common/test-utils';
 
-import useCKEditorCloud from '../../src/cloud/useCKEditorCloud';
+import useCKEditorCloud from '../../src/cloud/useCKEditorCloud.js';
 
 describe( 'useCKEditorCloud', () => {
 	beforeEach( removeAllCkCdnResources );
 
 	it( 'should load CKEditor bundles from CDN', async () => {
 		const { result } = renderHook( () => useCKEditorCloud( {
 			version: '43.0.0',
-			languages: [ 'en', 'de' ]
+			translations: [ 'en', 'de' ]
```

Comment:
```suggestion
			translations: [ 'es', 'de' ]
```

I'm not sure about this